server/src/middlewares/webAppProxyMiddleware.ts

```diff
@@ -38,7 +38,7 @@ const computeCanonicalUrl = (url: URL): URL => {
 }
 
 const addCanonicalLinkToHTML = (html: string, url: URL): string => {
-  return html.replace('<head>', `<head><link rel="canonical" href="${url}" />`)
+  return html.replace('<head>', `<head><link rel="canonical" href="${url.toString()}" />`)
```

Comment:
Pour fix `error Invalid type "URL" of template literal expression @typescript-eslint/restrict-template-expressions`